mafiasi/gprot/models.py

```diff
@@ -6,17 +6,38 @@
 
 from mafiasi.teaching.models import Course, Teacher
 
+
 def make_filename(obj, prefix, ext):
     return '{0}/{1}-{2}.{3}'.format(
         prefix, hexlify(os.urandom(16)), obj.pk, ext)
 
+
 def make_gprot_filename(gprot, filename):
     return make_filename(gprot, 'gprot', 'pdf')
 
+
 def make_attachment_filename(attachment, filename):
     ext = attachment.mime_type.split('/')[-1]
     return make_filename(attachment, 'gprot-attachment', ext)
 
+
+class LabelManager(models.Manager):
+
+    def as_choices(self):
+        for label in self.all().order_by('name'):
```

Comment:
There is no need for `.all()`, you can call `order_by()` on the manager directly without accessing `all()` first.

Reply:
For future reference this was copied from https://github.com/fsinfuhh/mafiasi/blob/65381497b80e5b7df9cccde3ecca077c650aa347/mafiasi/teaching/models.py and could be changed there in a separate pull request.